src/fielddef.ts

```diff
@@ -223,6 +224,12 @@ export interface PositionFieldDef<F> extends ScaleFieldDef<F> {
    * (3) At least one of non-position channels mapped to an unaggregated field that is different from x and y.  Otherwise, `null` by default.
    */
   stack?: StackOffset | null;
+  /**
+   * An object defining the properties of the Impute Operation to be applied.
+   * The field value of the other positional channel is taken as `key` of the `Impute` Operation.
+   * The field of the `color` channel if specified is used as `groupby` of the `Impute` Operation.
+   */
+  impute?: ImputeProperties;
```

Comment:
Please name this `ImputeParams`.

I think the docs here would look a bit out of place in the documentation.